src/_pytest/mark/structures.py

```diff
@@ -367,7 +368,7 @@ def __repr__(self):
         return "<NodeKeywords for node {}>".format(self.node)
 
 
-@attr.s(cmp=False, hash=False)
+@attr.s(hash=False, **attrs_no_eq)  # type: ignore
```

Comment:
Following the `ATTRS_EQ_FIELD` suggestion, this becomes `hash=False, **{ATTRS_EQ_FIELD: False}`, although I'm not sure if I actually prefer to have the `attrs_no_eq` dict in the module like you did here @hynek.